python/whylogs/viz/enums/enums.py

```diff
@@ -0,0 +1,17 @@
+from dataclasses import dataclass
+from enum import Enum
+
+
+@dataclass
+class PageSpec:
+    html: str
+    height: str
+
+
+class PageSpecEnum(Enum):
```

Comment:
Someday I'd love to break these out a bit differently so they are less copy and paste of each other, but not needed in this iteration.